client/src/client-only-routes/ShowCertification.js

```diff
@@ -123,7 +155,15 @@ const ShowCertification = props => {
         }
       });
     }
-  }, [isDonationDisplayed, props]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [
+    isDonationDisplayed,
+    props.userFetchState,
+    props.signedInUserName,
+    props.isDonating,
+    props.cert,
+    props.executeGA
+  ]);
```

Comment:
```suggestion
  });
```
Since `isDonationDisplayed` prevents repeated GA calls and `isEmpty(user)` does the same for api calls, can't we allow it to fire after each render?